packages/block-library/src/group/edit.js

```diff
@@ -60,6 +170,25 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 				/>
 			</InspectorAdvancedControls>
 			<TagName { ...blockProps }>
+				{ ( wideSize || contentSize ) && (
+					<style>
```

Comment:
I'm injecting the declarative styles inline, potentially, we could consider CSS variables, the problem is that I don't want them to cascade and CSS variables do cascade.